apertium-crh.crh.lexc

```diff
@@ -8,159 +9,176 @@
-%{i%}             !
-%{l%}             !
-%{s%}             !
-%{y%}             !
-%{j%}             !
-%{n%}             !
-
-%{e%}
-%{ö%}
-%{o%}
-%{ı%}
-%{b%}
-%{p%}
-
-%{☭%}
-%{ъ%}             ! force back-vowel harmony
-%{ь%}             ! force front-vowel harmony
-
-%-
-%% 
- 
-%>                !
-%+
+%<px1sg%>         ! First person singular	
+%<px2sg%>         ! Second person singular	
+%<px3sg%>         ! Third person singular	
+%<px3sp%>         ! Third person singular/plural	
+%<px3pl%>         ! Third person plural	
+%<px1pl%>         ! First person plural	
+%<px2pl%>         ! Second person plural
+
+%<p1%>            ! First person	
+%<p2%>            ! Second person 	
+%<p3%>            ! Third person
+
+%<sg%>            ! Singular  	
+%<pl%>            ! Plural 
+
+%<attr%>          ! Attributive	
+%<subst%>         ! Substantive	
+%<advl%>          ! Adverbial	
+
+%<pers%>          ! Personal	
+%<dem%>           ! Demonstrative	
+%<ind%>           ! Indefinite	
+%<itg%>           ! Interrogative	
+%<qnt%>           ! Quantifier
+%<ord%>           ! Ordinal	
+
+%<m%>             ! Masculine    	
+%<f%>             ! Feminine     	
+%<mf%>            ! Either
+
+%<cog%>           ! Cognomen (family name)	
+%<ant%>           ! Anthroponym (person name)	
+%<pat%>           ! Patronymic	
+%<org%>           ! Organisation	
+%<al%>            ! Others
+%<top%>           ! Toponym (place name)
+
+%<percent%>       ! Percent(%)	
+%<hash%>          ! Hash(#)
+
+%<sent%>          ! Sentence marker	
+%<guio%>          ! Hyphen	
+%<cm%>            ! Comma	
+%<apos%>          ! Apostrophe	
+%<rquot%>         ! Quote marker (right hand side)	
+%<lquot%>         ! Quote marker (left hand side)	
+%<rpar%>          ! Parenthetical marker (right hand side)	
+%<lpar%>          ! Parenthetical marker (left hand side)	
+%<err_orth%> 	
+%{G%}             !	
+%{D%}             !                             	
+%{A%}             !	
+%{U%}             !	
+%{E%}	
+%{C%}             !	
+%{I%}             ! Can only be unrounded: i, ı	
+%{Y%}             ! Can be rounded or unrounded: i, ü, ı, u	
+%{i%}             !	
+%{l%}             !	
+%{s%}             !	
+%{y%}             !	
+%{j%}             !	
+%{n%}             !	
+%{e%}	
+%{ö%}	
+%{o%}	
+%{ı%}	
+%{b%}	
+%{p%}	
+%{☭%}	
+%{ъ%}             ! force back-vowel harmony	
+%{ь%}             ! force front-vowel harmony	
+%-	
+%% 	
+ 	
+%>                !	
+%+		  !
+
+
+%<part%>          ! Particle 	
+%<emph%>          ! Emphatic [imperative]	
+%<cond%>          ! Conditional  ! Шарт  ! Условное  !! -с{A}		
+%<rec%>           ! Reciprocal (?)	
+%<iver%>	  ! suffix conveying ease/speed of action	
+%<eyaz%>	  ! suffix conveying the action *almost* happened	
+%<dub%>           ! dubative -mIş	
+%<abstr%>         ! Abstract suffix	
+%<nec%>           ! Necessitive	
+%<ref%>           ! Reflexive	
+%<recip%>         ! Reciprocal	
+%<coll%>          ! Collective	
+%<dist%>          ! Distibutive	
```

Comment:
Where did you get these from?  Some of them are duplicates, and some of them are not in the file....